server/src/main/java/io/openbytes/odi/application/ODIResourceService.java

```diff
@@ -0,0 +1,32 @@
+package io.openbytes.odi.application;
+
+import cn.hutool.core.util.StrUtil;
+import io.openbytes.odi.domain.ODIResource;
+import io.openbytes.odi.domain.storage.Storage;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.stereotype.Service;
+
+import java.util.Collections;
+import java.util.List;
+
+@Slf4j(topic = "metricsLog")
```

Comment:
why metricsLog?

Reply:
You can see log-back.xml, I just want to distinguish different log type, split into different log files.